declarations/npm Public Registry.json

```diff
@@ -3,7 +3,13 @@
   "documents": {
     "Privacy Policy": {
       "fetch": "https://docs.npmjs.com/policies/privacy",
-      "select": "#skip-nav"
+      "select": [
+        "main"
+      ],
+      "remove": [
+        "nav",
+        "details"
```

Comment:
This one is used on mobile. It was apparently [added on 26 Sept](https://github.com/OpenTermsArchive/demo-versions/commit/bff41a1206369a7843aac70f722d2cd5f05fab38) already, I suggest we add it to the history 🙂